### DISCUSSION TITLE
Does it support Arabic?

### DISCUSSION THREAD

**Discussion by kaabmassawi** [ocornut/imgui#6879] [New Users Build/Link/Run issues ONLY!]
I wonder if it supports arabic

**Comment by sakiodre** [comment]
Yes, you can use [custom glyph ranges](https://github.com/ocornut/imgui/blob/master/docs/FONTS.md#using-custom-glyph-ranges), but Dear ImGui doesn't support right-to-left layout, you might want to take a look at #4227 and [FarsiType](https://github.com/AmyrAhmady/FarsiType)

**Reply by ocornut** [reply]
Said thing is experimental and there’s no PR and it is quite unlikely this would work in a way satisfying performances requirements of dear imgui. So i would say unfortunately the answer is more of a No.

**Reply by kaabmassawi** [reply]
okay thanks for your reply
